src/Conversion/ONNXToKrnl/Math/Reduction.cpp

```diff
@@ -380,6 +448,10 @@ struct ONNXReduceSumOpLowering : public OpConversionPattern<ONNXReduceSumOp> {
     int64_t keepdims = adaptor.getKeepdims();
     bool isKeepdims = (keepdims == 1);
 
+    // Noop
+    auto noop = llvm::dyn_cast<ONNXReductionOp>(op).getNoopWithEmptyAxes();
```

Comment:
We don't have to cast here, `reduceOp` can be used directly, so `reduceOp. getNoopWithEmptyAxes()`;